index.js

```diff
@@ -3,6 +3,7 @@ const Discord = require('discord.js');
 const client = new Discord.Client();
 const rollRegex = /(^|\s+)[0-9]*(d|D)[0-9]+($|\s+)/g;
 const octaveRegex = /^_/g;
+const bofaRegex = \bBOFA\b/gi;
```

Comment:
da fuq is this amateur shit 

```suggestion
const bofaRegex = /\bBOFA\b/gi;
```